File: week05/week05-5.py
class Solution:
    def maxDistance(self, colors: List[int]) -> int:
        N=len(colors)
        ans=0
        for i in range(N):
            for j in range(N):
                if colors[i] != colors[j] and j-i > ans:
                    ans=j-i
        return ans

File: week05/week05-5.c
int maxDistance(int* colors, int colorsSize) {
    int ans=0;
    for(int i=0;i<colorsSize;i++){
        for(int j=0;j<colorsSize;j++){
            if(colors[i]!=colors[j] && j-i>ans){
                ans=j-i;
            }
        }
    }
    return ans;
}